proj/src/main/webapp/script.js

```diff
@@ -110,18 +111,23 @@ function initRef() {
             firstChildKey = snap.key;
         } else {
             const li = document.createElement('li');
-            li.innerText = snap.val();
+            li.innerText = snap.val().content;
             listObject.appendChild(li);
         }
     });
 }
 
 function pushChatMessage() {
-    const messageRef = firebase.database().ref(PATH);
-
     const messageInput = document.getElementById('message-input');
+
+    const chatRef = firebase.database().ref(PATH);
+
+    var message = {
+        content : messageInput.value,
+        timestamp : new Date().getTime()
+    }
```

Comment:
Cool, thanks for starting this. Once I have auth more up and running I'll add relevant user data to message

Reply:
^^ i am assuming we can also put profile img for each user in the future

Reply:
I'm not sure we'd want to store profile image here because then we'd be having an image stored with each message. I think maybe storing username and then calling the server to get the profile images for the username might help reduce the number of places you need to store the same data.

Reply:
^ what arinze said, i think we should store profile picture and display name with the user data instead of the message itself